go/test/endtoend/tabletgateway/buffer/buffer_test_helpers.go

```diff
@@ -246,6 +246,7 @@ func (bt *BufferingTest) createCluster() (*cluster.LocalProcessCluster, int) {
 	clusterInstance.VtGateExtraArgs = append(clusterInstance.VtGateExtraArgs, bt.VtGateExtraArgs...)
 
 	// Start vtgate
+	clusterInstance.VtGatePlannerVersion = 0
```

Comment:
Eeek. Why not use `planbuilder.V3` instead?

Reply:
`clusterInstance.VtGatePlannerVersion = 0` results in using VTGate's default, once we upgrade to `Gen4` as the default planner, we won't have to change this

Reply:
```go
if vtgate.PlannerVersion > 0 {
 	args = append(args, "-planner_version", vtgate.PlannerVersion.String())
}
```